### ISSUE TITLE
99 problems: change "tests" to "examples"

### ISSUE THREAD

**Issue by agarwal** [ocaml/v2.ocaml.org#176]
The [99 problems](http://ocaml.org/tutorials/99problems.html) page provides "tests". For example:

```
# last [] = None;;
- : bool = true
```

Although "tests" should be boolean conditions, don't we really want "examples" on this page. I think the following would be more intuitive:

```
# last [];;
-: bool option = None
```

**Comment by Chris00** [comment]
Right.  I think it stems from the prolog background.  We can change that.

**Comment by Chris00** [comment]
Fixed by 56dda4de7079fdd1bda8d0c47d4250ef2bddbacc
